src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv

```diff
@@ -1,15 +1,17 @@
-9.2.0,9.3.0,Transition notes
-12,12,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.
-Water to Water Heat Pump Load Side Heat Transfer Rate,Heat Pump Load Side Heat Transfer Rate,
-Water to Water Heat Pump Load Side Heat Transfer Energy,Heat Pump Load Side Heat Transfer Energy,
-Water to Water Heat Pump Source Side Heat Transfer Rate,Heat Pump Source Side Heat Transfer Rate,
-Water to Water Heat Pump Source Side Heat Transfer Energy,Heat Pump Source Side Heat Transfer Energy,
-Water to Water Heat Pump Load Side Inlet Temperature,Heat Pump Load Side Inlet Temperature,
-Water to Water Heat Pump Load Side Outlet Temperature,Heat Pump Load Side Outlet Temperature,
-Water to Water Heat Pump Source Side Inlet Temperature,Heat Pump Source Side Inlet Temperature,
-Water to Water Heat Pump Source Side Outlet Temperature,Heat Pump Source Side Outlet Temperature,
-Water to Water Heat Pump Electric Power,Heat Pump Electric Power,
-Water to Water Heat Pump Electric Energy,Heat Pump Electric Energy,
-Water to Water Heat Pump Load Side Mass Flow Rate,Heat Pump Load Side Mass Flow Rate,
-Water to Water Heat Pump Source Side Mass Flow Rate,Heat Pump Source Side Mass Flow Rate,
-old variable name,new variable name,-- add variable names (before this line) and leave off units -- <DELETE> to delete
+9.2.0,9.3.0,Transition notes - some of these are EMS variable names
+14,14,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.
+Water to Water Heat Pump Load Side Heat Transfer Rate,Heat Pump Load Side Heat Transfer Rate,
```

Comment:
@mitchute Not sure why github diff shows all of these lines have changed. Local diff does not. Only line 2 was change and lines 15 and 16 were added.

Reply:
Agreed, that's odd. Everything looks good #7730. Potentially different line endings or encoding, but both seem unlikely.